package.json

```diff
@@ -19,6 +21,7 @@
     "@testing-library/jest-dom": "^5.16.2",
     "@testing-library/react": "^12.1.4",
     "classnames": "2.3.1",
+    "date-fns": "^2.28.0",
```

Comment:
Is this package necessary? I'm not seeing it imported anywhere - is it a dependency of the mui/x date picker?

Reply:
Yes, all the date related packages added are dependencies of @mui/x date picker.